Makefile

```diff
@@ -1,6 +1,7 @@
 # Package configuration
 PROJECT = gitbase
 COMMANDS = cmd/gitbase
+GO_BUILD_ARGS += -tags mysql_go_regex
```

Comment:
is this needed?

Reply:
Without this, all binaries except the static will have oniguruma